sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets

```diff
@@ -50,8 +52,21 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <!-- 
   Build the webjobs extensions in ".azurefunctions"
   -->
+
+  <Target Name ="_WorkerExtensionsBuildCopy" AfterTargets="_WorkerExtensionsBuild">
+    <ItemGroup>
+      <ExtensionBinaries Include="$(ExtensionsCsProjFilePath)\buildout\bin\**\*.*"
+                         Exclude="$(ExtensionsCsProjFilePath)\buildout\bin\runtimes\**\*.*" />
+      <ExtensionRuntimeBinaries Include="$(ExtensionsCsProjFilePath)\buildout\runtimes\**\*.*" />
```

Comment:
I noticed that when you do `dotnet build` for extensions, the `<output>\bin` doesn't have all `runtimes` (linux, mac missing). But it's there in `<output>\runtimes`. 
So I am copying the entire `<output>\bin` except the `runtimes` directory. Instead getting the `runtimes` directory from output directly.